app/models/ncr/work_order.rb

```diff
@@ -193,19 +197,27 @@ def system_approver_emails
     end
 
     def self.ba61_tier1_budget_mailbox
-      ENV['NCR_BA61_TIER1_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com'
+      self.approver_with_role('BA61_tier1_budget_approver')
     end
 
     def self.ba61_tier2_budget_mailbox
-      ENV['NCR_BA61_TIER2_BUDGET_MAILBOX'] || 'communicart.ofm.approver@gmail.com'
+      self.approver_with_role('BA61_tier2_budget_approver')
+    end
+
+    def self.approver_with_role(role_name)
+      users = User.with_role(role_name).where(client_slug: 'ncr')
+      if users.empty?
+        fail "Missing User with role #{role_name} -- did you run rake db:migrate and rake db:seed?"
```

Comment:
Moreso the former. Perhaps I'm looking for an equivalent of what we do in env.rb, where we indicate optional and required env vars. We no longer have that visibility. Not a huge deal for this PR of course, but just something that I'm thinking about now that we've moved it out of the codebase. Maybe it's something we surface into the user management admin.